nginx-controller/nginx/configurator.go

```diff
@@ -101,20 +111,36 @@ func (cnf *Configurator) addOrUpdateMergableIngress(mergeableIngs *MergeableIngr
 
 	minions := mergeableIngs.Minions
 	for _, minion := range minions {
+
+		for _, blacklistAnn := range minionBlacklist {
```

Comment:
could we have a function for filtering minion annotations?

Reply:
would it be more efficient if we iterate over all minion annotations and look up if a particular annotation is in the black list using a map?

Reply:
I optimized it, since annotations are already in a map[string]string, I just look over all the blacklist items and if any are in the master they are removed. Looking through the blacklist since its small and there could be many annotations in an Ingress Resource.